src/main/java/seedu/address/model/ModelTrackermon.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.model;
+
+import java.nio.file.Path;
+import java.util.function.Predicate;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.model.show.Show;
+
+public interface ModelTrackermon {
+    /** {@code Predicate} that always evaluate to true */
+    Predicate<Show> PREDICATE_SHOW_ALL_SHOWS = unused -> true;
+
+    /**
+     * Replaces user prefs data with the data in {@code userPrefs}.
+     */
+    void setUserPrefs(ReadOnlyUserPrefsTrackermon userPrefs);
+
+    /**
+     * Returns the user prefs.
+     */
+    ReadOnlyUserPrefsTrackermon getUserPrefs();
+
+    /**
+     * Returns the user prefs' GUI settings.
+     */
+    GuiSettings getGuiSettings();
+
+    /**
+     * Sets the user prefs' GUI settings.
+     */
+    void setGuiSettings(GuiSettings guiSettings);
+
+    /**
+     * Returns the user prefs' address book file path.
```

Comment:
I realized while using this code for the v1.3. Comments were not updated here. Will change it for you.